### ISSUE TITLE
[FEATURE] Prefix for auto-import

### ISSUE THREAD

**Issue by loicbisiere** [LouisMazel/maz-ui#1183]
## Is your feature request related to a problem? Please describe

The default configuration of maz-ui for nuxt has all the auto-import activated by default. The issue is, I have already some composable with the same name (eg. `useToast`). 

## Describe the solution you'd like

Add a `autoImportPrefix` in the nuxt config, this way if `autoImportPrefix = "Maz"`, all the composables that are auto-imported will have the prefix "maz" (eg. `useMazToast`).

## Describe alternatives you've considered

The current "manual solution" is to deactivate all the auto-import and create composable wrappers for each maz composables, eg.

```ts
import { useToast as useMazToast } from 'maz-ui';

export { useMazToast };
```

**Comment by LouisMazel** [comment]
Hi @loicbisiere,

Do you want to use maz-ui's useToast or just the one you already have?

If not (and to be sure you knows), you can disable automatic import for each composable specifically (not all composables)

If you want use it and many others, it's true that it can be off-putting.

I keep this issue open to implement it in the next version, it's a really god idea, thanks 🙂

**Comment by loicbisiere** [comment]
Hi @LouisMazel , thanks for the quick answer.
The goal is to be able to use both, or at least to make those composables not colliding with potential existing ones by prefixing them. In my example, it seems a bit weird to have two toast composable, but for other composable, it could make sense to use one from another lib or internal code for a specific case and the one from maz in another case.

**Comment by LouisMazel** [comment]
I released a beta version with these new options in version 3.48.4-beta.0

```bash
npm install maz-ui@beta
// or npm install maz-ui@3.48.4-beta.0
```

You can test it and it will be released in the next latest version :)

**Comment by LouisMazel** [comment]
Released in [v3.49.0](https://github.com/LouisMazel/maz-ui/releases/tag/v3.49.0)
